kratos/includes/kratos_parameters.h

```diff
@@ -151,7 +152,7 @@ class Parameters {
   using iterator = iterator_adaptor;
   using const_iterator = const_iterator_adaptor;
 
-  Parameters(const std::string json_string) {
+  Parameters(const std::string& json_string = "{}") {
```

Comment:
mmm, passing a reference to a string used to be problematic from python (at least valgrind did not love that).

I don't know if this is changed with pybind

Reply:
It's done in the pybind examples, so it should be ok (https://pybind11.readthedocs.io/en/stable/classes.html?highlight=string%26)